src/book/01-team/04-history/01-team-history.mdx

```diff
@@ -38,3 +38,66 @@ NUbots made it to the quarter-finals of the competition. Overall we played 13 ga
 | Eliminations for Quarter-Finals (Penalty Shootout) | 3      | 1      | 1   | 0    | 0    |
 | Quarter-Finals                                     | 0      | 1      | 0   | 0    | 1    |
 | Drop-Ins                                           | -5     | 8      | 4   | 1    | 3    |
+
+## 2010: Singapore
+
+**Major hardware changes**
+
+- NAOv3: Intel ATOM 1.6GHz, 1GB Ram,
+
+**Major software changes**
+
+** Performance and Results**
+NUBots were unfortunately knocked out during second round robins, due to a loss with rUNSWift.
+
+| Round              | Points | Played | Won | Drew | Lost |
+| ------------------ | ------ | ------ | --- | ---- | ---- |
+| First Round Robin  | 6      | 2      | 2   | 0    | 0    |
+| Second Round Robin | 2      | 3      | 0   | 2    | 1    |
+
+## 2009: Graz, Austria
+
+**Major hardware changes**
+
+- Aldebaran NAOs get an upgrade (NAOv2), Dual Non-overlapping Cameras, so they can see their feet without having to bend over..
+
+**Major software changes**
+
+- Code Purge, Remove redudent AIBO Code realised by NUPlatform
+- New Git Repository, moved away from CVS
+- Introduction of Threading for NAOs, (SeeThinkThread: 30Hz, SenseMoveThread: 100Hz).
+- Blackboard based architecture
+- Soft Colour Classifications: A transition colour, that that can belong in different categories of objects, that is assigned an object with context at a later stage in the processing pipeline.
+- Advances in Line Detection, with the introduction of corner point (L and T), penalty spot, and center circle detection (ellipse fitting).
+- Improvements in Localisation as a direct result of the additional field objects seen.
+- Multiple Models Extended Kalman Filter
+- Goal Keeper: Right Dive, Left Dive, and Center
+
+** Performance and Results**
+NUBots were quarter-finalists in 2009.
+
+| Round          | Points | Played | Won | Drew | Lost |
+| -------------- | ------ | ------ | --- | ---- | ---- |
+| Round Robins   | 13     | 5      | 4   | 1    | 0    |
+| Quarter-Finals | 0:1    | 1      | 0   | 0    | 1    |
+
+## 2008: Suzhou, China
+
+**Major hardware changes**
+
+- First year with the Aldebaran NAOs (x86 AMD GEODE 500MHz CPU, 256 MB SDRAM / 2 GB flash memory)
+- Low-Powered Walk Optimisation
+
+**Major Software changes**
+
+- AIBO 2007 Code base to interface with NAOQI (Aldebarans Middleware for communicating with NAOs low-level hardware).
+
+** Performance and Results**
+NUIManoids (The University of Newcastle (NUbots) partnered with National University of Maynooth from Ireland) - First place, Standard Platform League.
```

Comment:
Let's highlight our win in bold 😅

```suggestion
NUIManoids (The University of Newcastle (NUbots) partnered with National University of Maynooth from Ireland) - **First place, Standard Platform League**.
```

Reply:
Also, was it NUImanoids or NUmanoids? The former makes sense for University of Ireland, but I think I've seen the latter before.

Reply:
https://www.newcastle.edu.au/__data/assets/pdf_file/0011/126002/NUbotFinalReport2008.pdf
Numanoids.

![image](https://user-images.githubusercontent.com/141890/107141901-c877f580-697f-11eb-8444-3ed24750cd35.png)